src/components/Header.jsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+import styles from '@/styles/Header.module.css';
+
+const Header = (props) => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+    color: '#aeadad',
+    textAlign: 'center',
+  };
+  return (
+    <header style={headerStyle} className={styles.header}>
+      {props.children}
+    </header>
```

Comment:
- [ ] you should use destruction in your code base kindly try to replace it as much as possible this will provide improved code readability  and sometimes we could have deep nested props in that place using destruction is helpfull